api/Worker.json

```
@@ -23,6 +23,15 @@
             "version_added": "10"
           },
           "nodejs": [
+            {
+              "version_added": "12.17.0",
+              "partial_implementation": true,
+              "notes": [
+                "Is a Node <code>EventEmitter</code> instead of DOM <code>EventTarget</code>.",
+                "Worker script environment expects CommonJS modules or ECMAScript2015 modules.",
```

The Node.js docs refer to these as "ECMAScript modules" and MDN as "modules". I think we should also avoid the spec year/number stuff too.

```suggestion
                "Worker script environment expects CommonJS modules or ECMAScript modules.",
```